tests/test_tiling_segmentation.py

```diff
@@ -37,7 +37,7 @@ def test_outputted_chips(self):
         tiling(cfg)
         # expected number of patches is constant due to random seed set in tiling script
         out_labels = [
-            (Path(f"{data_dir}/{proj}/trn/22978945_15_uint8_clipped/labels_burned"), 16),
+            (Path(f"{data_dir}/{proj}/trn/22978945_15_uint8_clipped/labels_burned"), 13),
```

Comment:
How come the output amount of labels is less than before if imagery has no nodata?